Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/InputActionsTreeViewItem.cs

```diff
@@ -102,16 +97,6 @@ async void DelayCall()
             renameTextfield.Q<TextField>().Focus();
         }
 
-        public void DeleteItem()
-        {
-            OnDeleteItem?.Invoke(0);
-        }
-
-        public void DuplicateItem()
-        {
-            OnDuplicateItem?.Invoke(0);
-        }
```

Comment:
this is ok in my point of view as long as it is still working the intended way - even though I don't understand the context to the problem of the PR